client/src/core/TLID.res

```diff
@@ -1,6 +1,9 @@
 module T = {
+  // WHAT is this?
   module Nested = {
-    @ppx.deriving(show({with_path: false})) type rec t = TLID(UInt64.t)
+    @ppx.deriving(show({with_path: false}))
```

Comment:
This shit is how you make a comparator 😬

Reply:
wow.